app/controllers/fp_users_controller.rb

```diff
@@ -16,13 +16,27 @@ def create
 
   def show
     @fp_user = FpUser.find(params[:id])
+    @reservations = Reservation.all.where(fp_user_id: current_user.id)
```

Comment:
このアクションは、下のどちらの用途で使うものでしょうか？

(A) FP が自分の予約を見るためのもの
(B) 普通のユーザーが FP の予約状況をみるためのもの

それによって current_user.id を使うかどうかが変わってくると思います。もし (A) の目的で用意している画面なら id は受け取らず 18行目は下のようにするのがよいと思います。

```diff
- @fp_user = FpUser.find(params[:id])
+ @fp_user = current_user
```

(B) の目的で用意している画面なら、その FP の予約状況を見るためのものなので下のようにするべきです。

```suggestion
    @reservations = Reservation.all.where(fp_user_id: @fp_user.id)
```

また、他のユーザーの予約が見えるべきではない、というふうに考えるのであれば下のようになるでしょう。

```suggestion
    @reservations = Reservation.all.where(fp_user_id: @fp_user.id, user_id: current_user.id)
```

Reply:
(B)の方で考えていました。
今見返すと、よくわからない中途半端な抽出になってしまっていました。
レビューありがとうございます！